apps/web-staking/src/app/components/redeem/History.tsx

```diff
@@ -187,6 +188,7 @@ export default function History({ redemptions, reloadRedemptions }: {
 		if (isError) {
 			updateOnError()
 		}
+		refreshEsXaiBalance();
 	}, [isSuccess, isError, updateOnSuccess, updateOnError]);
```

Comment:
Will this not need the refreshEsXaiBalance to be added to the dependencies array ?
I know we already have a bunch of warning for these missing dependencies but we should try to not add on top if that makes sense.